$NetBSD: patch-ba,v 1.2 2001/05/12 00:04:44 dmcmahill Exp $

The first change (the free line) had the following patch comment:

Index: x509.c
===================================================================
RCS file: /cvs/src/sbin/isakmpd/x509.c,v
retrieving revision 1.46
retrieving revision 1.47
diff -u -r1.46 -r1.47

the second (size_t -> u_int32_t) is to make the types match on 
64-bit machines

--- work.alpha/isakmpd/x509.c.orig	Tue Mar 27 11:50:35 2001
+++ work.alpha/isakmpd/x509.c	Thu May 10 17:02:00 2001
@@ -546,5 +546,4 @@
 	    }
 
-	  free (x509_policy_asserts);
 	  x509_policy_asserts = new_asserts;
 	}
@@ -626,5 +625,5 @@
   struct x509_hash *cert;
   u_int8_t **cid;
-  size_t *clen;
+  u_int32_t *clen;
   int n, i, id_found;
 
